test/rest_api/rest_util.py

```diff
@@ -30,3 +30,26 @@ def new_test_snapshot(rest_api, keyspaces=[], tables=[], tag=""):
     finally:
         resp = rest_api.send("DELETE", "storage_service/snapshots", params)
         resp.raise_for_status()
+
+@contextmanager
+def new_test_module(rest_api):
+    name = "test_module"
```

Comment:
I'm not sure what function the name plays here.
But it better match the internal name we use as hard coded (i.e. "test")